packages/react/src/Delay.tsx

```diff
@@ -3,15 +3,15 @@ import { DelayDefaultPropsContext } from './contexts'
 import { useTimeout } from './hooks'
 import { Delay_ms_prop_should_be_greater_than_or_equal_to_0, SuspensiveError } from './models/SuspensiveError'
 
-export interface DelayProps extends PropsWithChildren {
+export interface DelayProps {
   ms?: number
   /**
    * @experimental This is experimental feature.
    */
   fallback?: ReactNode
 }
 
-export const Delay = (props: DelayProps) => {
+export const Delay = (props: PropsWithChildren<DelayProps>) => {
```

Comment:
```suggestion
export const Delay = (props: DelayProps) => {
```

In my opinion, if this is named for meaning props of Delay, because Delay have children prop, we should use PropsWithChildren in DelayProps internally

We can check ErrorBoundaryProps contain type PropsWithChildren internally
https://github.com/bvaughn/react-error-boundary/blob/master/src/types.ts#L18C33-L26